src/styles/card.scss

```diff
@@ -0,0 +1,113 @@
+.card {
+  display: flex;
+  width: 200px;
+  height: 408px;
+  box-sizing: border-box;
+  border: 1px solid $color-border-card;
+  border-radius: 5px;
+  background-color: $color-bg-card;
+  flex-direction: column;
+
+  &__img {
+    margin-left: 19px;
+    margin-bottom: 40px;
+    margin-top: 32px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    display: flex;
+    margin-top: 0;
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-text-color;
+    width: 166px;
+    margin-left: 16px;
+
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    margin-top: 0;
+    font-size: 10px;
+    font-weight: 400;
```

Comment:
default value - no need to set it

Reply:
still not fixed
<img width="650" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/68aa21ac-3047-450f-8ad3-40731bdde443">

Reply:
don't working